pkgs/top-level/all-packages.nix

```diff
@@ -12465,6 +12467,7 @@ in
 
   abseil-cpp = callPackage ../development/libraries/abseil-cpp { };
 
+  accountsservice_0642 = callPackage ../development/libraries/accountsservice/0.6.42.nix { };
```

Comment:
```suggestion
  accountsservice_0_6 = callPackage ../development/libraries/accountsservice/0.6.nix { };
```

Reply:
Current version in Nixpkgs is 0.6.55, naming it 0.6.nix would suggest that 0.6.42 needs to be updated still.

Reply:
Didn't know that. ``accountsservice_0_6_42`` is probably still a bit more clear.